openlibrary/plugins/upstream/models.py

```diff
@@ -16,15 +18,14 @@
 from openlibrary.core import lending
 
 from openlibrary.plugins.search.code import SearchProcessor
+from openlibrary.plugins.upstream.utils import get_coverstore_url, MultiDict, parse_toc, get_edition_config
```

Comment:
I like this change because without the fully qualified import `pylint` would look into the wrong utils module.

Reply:
It seemed to be confusing the `utils` module available through `web.py` vs the utils local to that directory. Not super familiar with how python prioritizes this, but seemed to grab the correct one when run locally.